eventtracking/handlers.py

```diff
@@ -0,0 +1,61 @@
+"""
+This module contains the handlers for signals emitted by the analytics app.
+"""
+import json
+import logging
+
+from django.dispatch import receiver
+from openedx_events.analytics.signals import TRACKING_EVENT_EMITTED
+from openedx_events.tooling import SIGNAL_PROCESSED_FROM_EVENT_BUS
+
+from eventtracking.backends.event_bus import EventBusRoutingBackend
+from eventtracking.processors.exceptions import EventEmissionExit
+from eventtracking.tracker import get_tracker
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(TRACKING_EVENT_EMITTED)
```

Comment:
If we're not checking for whether the event is from the event bus, wouldn't this result in processing the event in-process as a normal Django signal handler as well as pushing it to the bus?

Reply:
Yes, that's why we are checking the argument here:

```python
    if not kwargs.get(SIGNAL_PROCESSED_FROM_EVENT_BUS, False):
        logger.debug("Event received from a non-event bus backend, skipping...")
        return
```